channels-rke2.yaml

```diff
@@ -1373,7 +1373,7 @@ releases:
     featureVersions: *featureVersions-v1
   - version: v1.25.13+rke2r1
     minChannelServerVersion: v2.7.2-alpha1
-    maxChannelServerVersion: v2.8.99
+    maxChannelServerVersion: v2.9.99
```

Comment:
looks like when we moved from v2.7 to v2.8 we kept the maxVersion of some previously released versions to v2.7.99: 
https://github.com/rancher/kontainer-driver-metadata/blob/8d81713847c3f609f7d5a6b821386979f0e0dcb5/channels-rke2.yaml#L1351-L1353
so just wanted to confirm if it is okay to update maxVersion for each k8s version from v2.8.99 to v2.9.99 ?